src/main/java/core/basesyntax/report/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.report;
+
+import core.basesyntax.storage.Storage;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
```

Comment:
According to the checklist, classes should not begin with an empty line. Please remove the empty line before the class declaration.